README.md

```diff
@@ -16,11 +16,57 @@ To install, simply add it to your `composer.json` file:
 $ composer require myonlinestore/omnipay-klarna-checkout
 ```
 
+## Initialization
+
+```php
+$gateway = Omnipay::create('\MyOnlineStore\Omnipay\KlarnaCheckout\Gateway');
+
+$gateway->initialize(['username' => $username, 'secret' => $secret]);
+// or 
+$gateway->setUsername($username);
+$gateway->setSecret($secret);
+```
+
 ## Usage
 
 For general usage instructions, please see the main [Omnipay](https://github.com/thephpleague/omnipay)
 repository.
 
+To create a new order, use the `authorize` method:
+```php
+$data = [
+    'amount'           => 100,
+    'tax_amount'       => .2,
+    'currency'         => 'SEK',
+    'locale'           => 'SE',
+    'purchase_country' => 'SE',
+    
+    'notify_url' => '', // https://developers.klarna.com/api/#checkout-api__ordermerchant_urls__validation
```

Comment:
I believe these merchant URL should all be camel cased

Reply:
As far as I can tell they all get converted into camel case and validated correctly when you provide snake case keys. I tried to keep the examples in line with the Klarna docs, hence the snake case.